docs/shared/getting-started/installation.md

```diff
@@ -90,4 +90,121 @@ pnpm install --global nx@latest
 
 The advantage of a global installation is that you don't have to prefix your commands with npx, yarn or pnpm. The global Nx installation hands off the process execution to the local Nx installation in your repository, which eliminates any issues with outdated globally installed packages.
 
-Learn more about [managing and troubleshooting a global Nx installation](/concepts/more-concepts/global-nx).
+### Updating your global Nx installation
```

Comment:
I partially agree. The thing is no one finds it hidden somewhere while everyone searches installation related things in the "Installation" doc 🤷. Before it was somewhere hidden in the "More concepts" page. I could see this being a dedicated recipe because it is an edge case, but then we'd move it into "tips & tricks" where it would be a bit lost

Reply:
I'm fine with adding a link to the recipe from this page.  That way people can find it.  But it shouldn't be fully described here.